clients/src/main/java/org/apache/kafka/clients/consumer/internals/Coordinator.java

```diff
@@ -217,21 +217,6 @@ public void ensureCoordinatorKnown() {
         }
     }
 
-    /**
```

Comment:
I think the comment was old. Previously we blocked when the coordinator wasn't ready, but now we just fail the request to keep it asynchronous.